src/watchful/client.py

```diff
@@ -1081,6 +1081,49 @@ def export_preview(mode: str = "ftc") -> Optional[Dict]:
     return api("export_preview", mode=mode)
 
 
+def is_utf8(
+    filepath: str,
+    threshold: float = 0.5,
+    is_fast: bool = True,
+) -> Literal[-1, 0, 1]:
```

Comment:
At first glance, a `boolean `true` return when we have confidence the file is UTF-8 and `false` when we either don't meet the confidence threshold or the file is definitively not UTF-8 seems to me a more expressive signature. Another option could be to raise an error or exception when the threshold isn't met, but that doesn't seem necessary.

Reply:
I see where you're going, your suggestion is more intuitive for users. I agree, I think at this point there is no need to explicitly differentiate "don't meet the confidence threshold" and "the file is not utf8".